cohortextractor/tpp_backend.py

```diff
@@ -2548,6 +2549,9 @@ def patients_admitted_to_hospital(
             value_sql = ", ".join(map(quote, to_list(column_value)))
             conditions.append(f"{supported_columns[column_name]} IN ({value_sql})")
 
+        if with_at_least_one_day_in_critical_care:
+            conditions.append("APCS_Der.Spell_PbR_CC_Day > 0")
```

Comment:
This is a bit superfluous because MSSQL will spot that we're comparing to an int and CAST the string column automatically, but it might be clearer to do this explicitly e.g.
```suggestion
            conditions.append("CAST(APCS_Der.Spell_PbR_CC_Day AS int) > 0")
```

Reply:
Good point -  while I was adding the tpp_backend code I'd thought about that and meant to come back to it, but I was kind of expecting the tests to compain and remind me :)